components/layermanager/layermanager-gallery.component.ts

```diff
@@ -58,4 +61,18 @@ export class HsLayerManagerGalleryComponent {
         450
     );
   }
+  expandMenu(e, layer) {
+    const parent = (e.target as Element).parentElement;
+    this.HsLayerManagerService.toggleLayerEditor(
+      layer,
+      'settings',
+      'sublayers'
+    );
+
+    if (parent.classList.contains('expanded')) {
```

Comment:
can't we do this with ngClass instead? Is HsLayerManagerService.menuExpanded the same as HsLayerManagerGalleryComponent.menuExpanded and could we just use the variable in component since it deals mostly only with UI state.

Reply:
Its there :) 
https://github.com/hslayers/hslayers-ng/pull/1138/commits/476d99605d9420727519f9b0a84970beca384f99#diff-a616dc4693b080d259bac93af2291d92R72